lib/jekyll/plugin_manager.rb

```diff
@@ -29,6 +30,15 @@ def require_gems
       )
     end
 
+    def require_theme_deps
+      if site.theme.runtime_dependencies
+        site.theme.runtime_dependencies.each do |dep|
```

Comment:
I gez I'll go with the explicit `return` style instead of the alternative wherein a block is called even on an empty array.